Source/Pages/Gallery/YPLibraryView.swift

```diff
@@ -91,6 +91,22 @@ internal final class YPLibraryView: UIView {
         return v
     }()
 
+    public let bulkUploadRemoveAllButton: UIButton = {
+        var container = AttributeContainer()
+        container.font = YPConfig.fonts.bulkUploadCountFont
+        container.foregroundColor = UIColor.ypLabel
+        var configuration = UIButton.Configuration.plain()
+        configuration.attributedTitle = AttributedString("", attributes: container)
```

Comment:
I think this should have either a hidden title or an accessibility label, for accessibility purposes.